### DISCUSSION TITLE
What is the Variable axes as Weight?

### DISCUSSION THREAD

**Discussion by SwedenFontDesigner** [tonsky/FiraCode#1336] [Q&A] [answered]
Since the Weight same as **427**, What is The **recommended** Variable axes as Weight?

**Comment by tonsky** [comment, marked as answer]
Light 300
Regular 400
Retina 450
Medium 500
SemiBold 600
Bold 700

**Reply by SwedenFontDesigner** [reply]
Can you name the weight of **630**?

**Reply by tonsky** [reply]
There’s no dedicated name for it

**Reply by SwedenFontDesigner** [reply]
Ok then 👍🏼
